src/status_im/ui/screens/activity_center/notification/contact_request/view.cljs

```diff
@@ -0,0 +1,58 @@
+(ns status-im.ui.screens.activity-center.notification.contact-request.view
+  (:require [quo.components.animated.pressable :as animation]
+            [quo2.core :as quo2]
+            [status-im.constants :as constants]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.multiaccounts.core :as multiaccounts]
+            [status-im.ui.screens.activity-center.notification.contact-request.style :as style]
+            [status-im.ui.screens.activity-center.utils :as activity-center.utils]
+            [status-im.utils.datetime :as datetime]
+            [status-im.utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [id] :as notification}]
+  (let [message   (or (:message notification) (:last-message notification))
```

Comment:
Can we destructure message and `last-message` from `message` on line 13 ?

Reply:
I know you're a _destructuring lover_ from your reviews :heart_decoration: I'm going to change the code ;)